### PR TITLE
Tiff import

### LINKED ISSUE #31: Allow app to use TIFF stacks
Currently tiff stacks can be viewed but we have not yet implemented downsampling them. To run the DVC code we would need to convert the tiff to a numpy/raw file.

### PR DESCRIPTION
**Pull request by paskino**
Enable TIFF import

- [x] CILViewer TIFF resample reader https://github.com/vais-ral/CILViewer/pull/252
- [x] CILViewer TIFF cropped reader https://github.com/vais-ral/CILViewer/issues/126
- [x] https://github.com/TomographicImaging/iDVC/issues/104

closes #31

### CODE REVIEW

**Comment by paskino** [comment]
Before merging this one, do we need a release of the viewer?

**Comment by lauramurgatroyd** [comment]
yes

File: src/idvc/dvc_runner.py [hunk 1]
```diff
@@ -150,6 +151,8 @@ def finished_run(main_window, exitCode, exitStatus, process = None, required_run
     # print("did")
 
 class DVC_runner(object):
+    import pysnooper
+    @pysnooper.snoop()
```
Comment (lauramurgatroyd) [inline src/idvc/dvc_runner.py:155]:
```suggestion
```

**Comment by lauramurgatroyd** [comment]
I am happy it works well on windows.
I tested with tiffs but also numpy and raw to check they still work.

**Comment by lauramurgatroyd** [comment]
On linux I had various issues: #112  but I doubt they're related to this PR

**Comment by lauramurgatroyd** [comment]
This is waiting on viewer version 22.2
Milestone for that version is here: https://github.com/vais-ral/CILViewer/milestone/5

File: src/idvc/dvc_runner.py (hunk 1 above)
Comment (lauramurgatroyd) [inline src/idvc/dvc_runner.py:155]:
```suggestion
```

**Pull request merged**